src/ray/core_worker/store_provider/plasma_store_provider.h

```diff
@@ -76,7 +79,7 @@ class CoreWorkerPlasmaStoreProvider : public CoreWorkerStoreProvider {
   /// \param[in] num_attemps The number of attempted times.
   /// \param[in] unready The unready objects.
   static void WarnIfAttemptedTooManyTimes(
-      int num_attempts, const std::unordered_map<ObjectID, int> &unready);
+      int num_attempts, const std::unordered_map<ObjectID, std::vector<int>> &unready);
```

Comment:
The documentation here could be updated to mention that only the keys in the map are actually used.